src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicNoBlackHole.hpp

```diff
@@ -44,19 +41,18 @@ struct EvolutionMetavars
       tmpl::flatten<tmpl::list<
           Parallel::PhaseActions<Parallel::Phase::Initialization,
                                  initialization_actions>,
-          tmpl::conditional_t<
-              use_numeric_id,
-              tmpl::list<Parallel::PhaseActions<
-                             Parallel::Phase::RegisterWithElementDataReader,
-                             tmpl::list<importers::Actions::
-                                            RegisterWithElementDataReader,
-                                        Parallel::Actions::TerminatePhase>>,
-                         Parallel::PhaseActions<
-                             Parallel::Phase::ImportInitialData,
-                             tmpl::list<gh::Actions::ReadNumericInitialData,
-                                        gh::Actions::SetNumericInitialData,
-                                        Parallel::Actions::TerminatePhase>>>,
-              tmpl::list<>>,
+          Parallel::PhaseActions<
+              Parallel::Phase::RegisterWithElementDataReader,
+              tmpl::list<importers::Actions::RegisterWithElementDataReader,
+                         Parallel::Actions::TerminatePhase>>,
+          Parallel::PhaseActions<
+              Parallel::Phase::ImportInitialData,
+              tmpl::list<
+                  gh::Actions::SetInitialData,
+                  tmpl::conditional_t<VolumeDim == 3,
+                                      gh::Actions::ReceiveNumericInitialData,
+                                      tmpl::list<>>,
```

Comment:
Why only 3?

Reply:
Numeric ID is currently only implemented for 3D, mostly because we only need 3D and template parameters make some things harder (inheriting constructors, for example).

Reply:
Ok sounds good.